forta-mint-mainnet/src/agent.js

```diff
@@ -13,6 +13,8 @@ function provideHandleTransaction() {
     const filteredTxEvent = txEvent.filterLog(event, contractAddress);
 
     filteredTxEvent.forEach((tx) => {
+      const { from } = tx;
+      const fromToLower = from.toLowerCase();
```

Comment:
I got "Cannot read property 'toLowerCase' of undefined" when I tried the test transaction from the README. I think the problem is that the event doesn't have "from" property. Maybe you meant "address"?